platform/darwin/src/MGLGeometry.h

```diff
@@ -45,6 +45,18 @@ typedef struct MGLCoordinateBounds {
     CLLocationCoordinate2D ne;
 } MGLCoordinateBounds;
 
+typedef struct MGLCoordinateQuad {
+    /** Coordinate at the northwest corner. */
+    CLLocationCoordinate2D tl;
```

Comment:
Let's write out these field names. It was a mistake to abbreviate the MGLCoordinateBounds fields.